cypress/e2e/yves/core/locale-switching.cy.ts

```diff
@@ -52,6 +52,28 @@ describe('locale switching', { tags: ['@core', '@yves'] }, (): void => {
     }, catalogPage);
   });
 
+  it('should maintain locale when navigating to New page after switching locale.', (): void => {
+    homePage.visit();
+    homePage.getAvailableLocales();
+
+    const initialLocale = staticFixtures.localeEN;
+    const oppositeLocale = staticFixtures.localeDE;
+
+    localeSwitchingScenario.execute({
+      currentLocale: initialLocale,
+      selectedLocale: oppositeLocale,
+    });
+
+    cy.get('.navigation-multilevel-node .navigation-multilevel-node__link--lvl-1')
+      .contains(oppositeLocale === staticFixtures.localeDE ? 'Neu' : 'New')
+      .click({ force: true });
+
+    cy.get('[data-qa="component language-switcher"]').should(
+      'contain',
+      oppositeLocale === staticFixtures.localeDE ? 'DE' : 'EN'
```

Comment:
should we also check if the page is actually translated before we click and after we navigate to the new page? I mean that locale in the url is preserved and any one element on the page is in the correct language?